src/pages/less/constants.less

```diff
@@ -0,0 +1,6 @@
+@dark-blue: #1C4863;
```

Comment:
It's good practice to stay away from color in the variable name.  Something like `@brand-primary` is more maintainable.  See bootstrap variables.